common/djangoapps/django_sudo/utils.py

```diff
@@ -0,0 +1,32 @@
+"""
+sudo.utils
+"""
+import django.contrib.sessions.middleware
+import sudo.middleware
+
+
+def region_name(region):
```

Comment:
Our region is edx-specific, but the operation to make the region safe to put in a cookie isn't edx-specific. If you moved this functionality to nearer to the cookie storage, then a) it would be clearer, and b) you wouldn't need to call it every time you specify a region.